src/App.js

```diff
@@ -1,5 +1,133 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+
 class App {
-  async play() {}
+  async play() {
+    Console.print('숫자 야구 게임을 시작합니다.');
+
+    while (true) {
+      await this.gameFlow();
+      const userRestartInput = await this.getUserRestartInput();
+      if (userRestartInput !== 1) {
+        break;
+      }
+    }
+  }
+  // 게임 실행 흐름을 책임지는 함수
+  async gameFlow() {
+    const COMPUTER_ANSWER = this.createComputerAnswer();
+    let isUserWon = false;
+
+    while (!isUserWon) {
+      isUserWon = await this.checkUserGuess(COMPUTER_ANSWER);
+    }
+    return;
+  }
```

Comment:
저도 비슷한 의견입니다! 일반적으로 gameFlow()라는 이름을 보면 첫 시작문구 출력을 제외한 
**매 판(숫자생성 ~ 숫자 맞추기) 을 진행 시키도록 하실 의도의 함수**이지 않을까 생각이 드는데,

그렇다면 유저에게 재시작 여부를 묻는 `getUserRestartInput` 부분은 
gameFlow 아래로 함께 추상화해주시는게 좋지 않을까 생각했습니다!

```suggestion
  async gameFlow() {
    while (true) {
      const COMPUTER_ANSWER = this.createComputerAnswer();
      let doesUserWin = false;
     
      do { // 우선 처음에 반드시 어떤 명령(블럭)을 처리하고 나서 조건을 확인해야한다면, do-while 문을 써보시는 것도 추천드려요!! 
        doesUserWin = await this.checkUserGuess(COMPUTER_ANSWER);
      } while (!doesUserWin);

      const userRestartInput = await this.getUserRestartInput();
      if (userRestartInput === 2) { // 부정표현 보다는 긍정표현 사용하면 더 읽기 쉬운 것 같아요!
         break;
      }
    }
  }
```